mix.exs

```diff
@@ -97,4 +104,20 @@ defmodule Membrane.RTC.Engine.MixProject do
     Mix.shell().info(result)
     if exit_status != 0, do: raise("Failed to generate TS docs")
   end
+
+  defp packages_installed?() do
+    System.cmd("npm", ["ls", "--prefix", "assets", "--prefer-offline"], stderr_to_stdout: true)
+    |> case do
+      {output, 0} ->
+        missing =
+          output |> String.split("\n") |> Enum.filter(&Regex.match?(~r/UNMET DEPENDENCY/, &1))
```

Comment:
```suggestion
          output |> String.split("\n") |> Enum.filter(&Regex.match?(~r/UNMET DEPENDENCY|empty/, &1))
```
When I added this it works perfectly

Reply:
I added this to the changes.

Out of curiosity, do you have reproducible steps to get this message out of `npm ls`? I haven't seen it.

Reply:
Sure, when I deleted `node_modules` I get something like this (I am pasting the whole flow)
![image](https://user-images.githubusercontent.com/24651552/136055160-258071d4-dbf2-4b76-93e8-f633a2f37692.png)